core/persistence-jpa/src/main/java/org/apache/syncope/core/persistence/jpa/dao/repo/GroupRepoExt.java

```diff
@@ -50,6 +51,8 @@ public interface GroupRepoExt extends AnyRepoExt<Group> {
 
     List<UMembership> findUMemberships(Group group);
 
+    List<UMembership> findUMemberships(Group group, Pageable pageable);
```

Comment:
Please remove the existing  method

```java
List<UMembership> findUMemberships(Group group);
```

and replace all existing invocations

```findUMembership(group)```

with

```findUMembership(group, Pageable.unpaged())```

Naturally, you will need to adjust the new code to handle the case when `Pageable.unpaged()` is passed as an argument.